From: Christopher Huhn <c.huhn@gsi.de>
Date: Wed, 12 Mar 2014 11:55:51 +0100
Subject: TTL and IP are reverted in resolver response

---
 Slim/Networking/Async/DNS.pm |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Slim/Networking/Async/DNS.pm b/Slim/Networking/Async/DNS.pm
index d91c03d..7cf710f 100644
--- a/Slim/Networking/Async/DNS.pm
+++ b/Slim/Networking/Async/DNS.pm
@@ -109,8 +109,8 @@ sub resolve {
 			return;
 		}
 		
-		my $addr = $res->[3];
-		my $ttl	 = $res->[4];
+		my $addr = $res->[4];
+		my $ttl	 = $res->[3];
 		
 		main::DEBUGLOG && $log->is_debug && $log->debug( "Got DNS response $addr for $host (ttl $ttl)" );
 		
@@ -139,4 +139,4 @@ sub cached {
 	return;
 }
 
-1;
\ No newline at end of file
+1;
