From: Christopher Huhn <c.huhn@gsi.de>
Date: Tue, 11 Mar 2014 20:27:28 +0100
Subject: Don't attach in on_connect_do

---
 Slim/Utils/SQLiteHelper.pm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Slim/Utils/SQLiteHelper.pm b/Slim/Utils/SQLiteHelper.pm
index b62d835..b03e18d 100644
--- a/Slim/Utils/SQLiteHelper.pm
+++ b/Slim/Utils/SQLiteHelper.pm
@@ -143,7 +143,7 @@ sub on_connect_do {
 	# we need to migrate long 7.6.0 file names to shorter 7.6.1 filenames: Windows can't handle the long version
 	_migrateDBFile($class->_dbFile('squeezebox-persistent.db'), $persistentdb);
 
-	push @{$sql}, "ATTACH '$persistentdb' AS persistentdb";
+	#push @{$sql}, "ATTACH '$persistentdb' AS persistentdb";
 	push @{$sql}, 'PRAGMA persistentdb.journal_mode = WAL';
 	
 	return $sql;
