From: Christopher Huhn <c.huhn@gsi.de>
Date: Sat, 15 Mar 2014 18:26:35 +0100
Subject: Don't break recent DBIx::Class versions

---
 Slim/Schema.pm |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/Slim/Schema.pm b/Slim/Schema.pm
index 543c834..9fa1aed 100644
--- a/Slim/Schema.pm
+++ b/Slim/Schema.pm
@@ -294,8 +294,9 @@ sub _connect {
 	
 	my ($driver, $source, $username, $password) = $class->sourceInformation;
 
-	# For custom exceptions
-	$class->storage_type('Slim::Schema::Storage');
+	# For custom exceptions 
+	# FIXME: breaks DBIx::Class newer than 0.08112
+	#$class->storage_type('Slim::Schema::Storage');
 	
 	my $sqlHelperClass = Slim::Utils::OSDetect->getOS()->sqlHelperClass();
 	my $on_connect_do = $sqlHelperClass->on_connect_do();
