From: Christopher Huhn <c.huhn@gsi.de>
Date: Sun, 26 Nov 2017 22:40:23 +0100
Subject: Skip perl module check in favor of package management

---
 Slim/bootstrap.pm | 25 -------------------------
 1 file changed, 25 deletions(-)

diff --git a/Slim/bootstrap.pm b/Slim/bootstrap.pm
index 3a2dcca..bb0a70b 100644
--- a/Slim/bootstrap.pm
+++ b/Slim/bootstrap.pm
@@ -225,31 +225,6 @@ don't work for you, ask for help and/or report a bug.
 	# And we're done with the trying - put our CPAN path back on @INC.
 	unshift @INC, @SlimINC;
 	
-	# Check that all of our CPAN modules are the correct minimum version
-	my $failed = check_valid_versions();
-	if ( scalar keys %{$failed} ) {
-	
-		print "The following CPAN modules were found but cannot work with Logitech Media Server:\n";
-		
-		for my $module ( sort keys %{$failed} ) {
-			if ( $failed->{$module}->{loaded} eq $failed->{$module}->{need} && $failed->{$module}->{msg} ) {
-				print "  $module:\n" . $failed->{$module}->{msg} . "\n";
-			}
-			else {
-				print "  $module (loaded " . $failed->{$module}->{loaded} . ", need " . $failed->{$module}->{need} . ")\n";
-			}
-		}
-		
-		print "\n";		
-		print "To fix this problem you have several options:\n";
-		print "1. Install the latest version of the module(s) using CPAN: sudo cpan Some::Module\n";
-		print "2. Update the module's package using apt-get, yum, etc.\n";
-		print "3. Run the .tar.gz version of Logitech Media Server which includes all required CPAN modules.\n";
-		print "\n";
-		
-		exit;
-	}
-	
 	# Load PerfMon if enabled
 	if ( main::PERFMON ) {
 		require Slim::Utils::PerfMon;
